stable/README.md

```diff
@@ -1,196 +1,196 @@
-### The instructions on this page are in three parts:
```

Comment:
Why does this entire file show as being changed?

Reply:
Sure. I will make sure to remove prettier and format only files that require those changes.